src/atomate2/abinit/schemas/calculation.py

```diff
@@ -237,12 +291,42 @@ def from_abinit_files(
         abinit_gsr_file = dir_name / abinit_gsr_file
         abinit_log_file = dir_name / abinit_log_file
         abinit_abort_file = dir_name / abinit_abort_file
+        abinit_out_file = dir_name / abinit_out_file
+        abinit_outddb_file = dir_name / abinit_outddb_file
+        abinit_outpot_file = dir_name / abinit_outpot_file
+
+        abinit_objects: dict[AbinitObject, Any] = {}
+        if files_to_store is None:
+            files_to_store = ["DDB"]
```

Comment:
I don't think that the default should be to store the DDB files. As mentioned in the other comment the GS calculations also produce a DDB file and I am pretty sure that we don't want them to be stored for a relaxation or a band structure. The default should be no file saved (or at least files that would make sense to store for all the kind of calculations, but I would leave it empty at the moment). Then each kind of job should override this behaviour to store the files needed for that specific job. And it would be better to have the option for the user to customize this as well.
Can't the list of files_to_store be set in the ResponseMaker, selecting the DDB?

Reply:
I put the files_to_store = ['DDB'] as default in the task_document_kwargs of the ResponseMaker and let an empty list if it is None which is the default for the BaseAbinitMaker